rclc_lifecycle/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ find_package(std_msgs REQUIRED)
 find_package(lifecycle_msgs REQUIRED)
 find_package(rcl_lifecycle REQUIRED)
 find_package(rclc REQUIRED)
-
+find_package(Threads REQUIRED)
```

Comment:
Same here about micro-ROS if we want to build `rclc_lifecycle` in embedded platforms

Reply:
resolved in https://github.com/ros2/rclc/pull/237/commits/da1cebeea58447ff216323994f11e649bdd1e3a4